frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -48,12 +47,12 @@ const InfoHeader = styled.div`
 
 const InfoMain = styled.div`
   width: 100%;
-  max-height: 420px;
   padding-left: 10px;
   padding: 0px 10px 5px 10px;
 
   &.open {
     overflow-y: scroll;
+    max-height: calc(100vh - 380px);
```

Comment:
This allows the slide out to stretch to the edge of the screen if need be, but shrink for smaller screens

Reply:
does this take into account the height of the footer?

Reply:
yes, I added up (or tried to get close to) the header, filter bar and footer